run.py

```diff
@@ -421,7 +434,7 @@ def process_quiz(candidate_id, quiz_id):
     print(f"candidate id is {candidate_id}")
 
     score = right / total_questions
-    new_results = QuizResults(quiz_id, candidate_id, wrong, right, score)
+    new_results = QuizResults(quiz_id, candidate_id, right, wrong, time_elapsed, score)
```

Comment:
the only thing I'm not sure about here is swapping the QuizResults list of attributes here. Somehow the right and wrong vars got swapped and i'm worried that will impact the outcome. I will approve and then test and let you know!

Reply:
the right and wrong counts were switched to test actual correct answers to make stats correct

Reply:
perfect! thanks for looking!